jsk_perception/src/apply_mask_image.cpp

```diff
@@ -113,12 +125,22 @@ namespace jsk_perception
       cv::bitwise_not(mask, mask);
     }
 
+    cv::Rect region = jsk_recognition_utils::boundingRectOfMaskImage(mask);
+    if (camera_info_){
+      sensor_msgs::CameraInfo roi(*camera_info_);
+      roi.roi.x_offset = region.x;
+      roi.roi.y_offset = region.y;
+      roi.roi.width = region.width;
+      roi.roi.height = region.height;
+      roi.roi.do_rectify = true;
```

Comment:
This is just a question, but do we need to set this as true and can we set this as true?
```
# True if a distinct rectified ROI should be calculated from the "raw"
# ROI in this message. Typically this should be False if the full image
# is captured (ROI not used), and True if a subwindow is captured (ROI
# used).
```

Reply:
I set true without thinking too much. I think It is better to give true or false with rosparam whether to use rectified image as input. How about it?